Kinvey/Kinvey/HttpResponse.swift

```diff
@@ -57,6 +57,13 @@ struct HttpResponse: Response {
     var etag: String? {
         return response.allHeaderFields["Etag"] as? String
     }
+    
+    var contentTypeIsJson: Bool {
```

Comment:
nope, this was one of the changes that I made to try a few things, going to remove that